src/Gui/propertyeditor/PropertyItem.cpp

```diff
@@ -1376,9 +1376,11 @@ void PropertyBoolItem::setValue(const QVariant& value)
 }
 
 QWidget* PropertyBoolItem::createEditor(QWidget* parent,
-                                        const std::function<void()>& method,
+                                        const std::function<void()>& /*method*/,
```

Comment:
Why aren't we using `[[maybe_unused]]` as in other places?

Reply:
Because it's not "maybe" unused - it's *definitely* unused. We use [[maybe_unused]] when one branch of a preprocessor ifdef would cause the variable to be unused, for example.

Reply:
Related discussion https://discord.com/channels/870877411049357352/1354091047185154058/1377973057515425843